### DISCUSSION TITLE
Do i have to PR to public repository to get `Pull Shark` badge?

### DISCUSSION THREAD

**Discussion by Yangeok** [community/community#19482] [Other Feature Feedback, Questions, & Ideas] [answered]
Can't I get a badge if I PR in own private repository?

**Comment by melvincwng** [comment, marked as answer]
@Yangeok, short answer - yes it has to be in a public repo.

Hope this clarifies.

**Reply by melvincwng** [reply]
@Yangeok, for more information on achievements, please refer to [this link](https://github.com/Schweinepriester/github-profile-achievements).

If this answer is helpful, please 'Mark as answer'.

Thanks, and have a great day 😄

**Reply by Yangeok** [reply]
Thank you for answer. Have a good one :)

**Reply by melvincwng** [reply]
@Yangeok, pleasure to have helped ;)

If you would like to stay in touch, kindly follow me on GitHub.

Otherwise, thank you for marking this reply as answer. 

Appreciate the help and support provided ^^

**Comment by RonaldGuilhermePDS** [comment]
https://github.com/Schweinepriester/github-profile-achievements
